v4/source/roles.yaml

```
@@ -1,3 +1,37 @@
+  "/roles":
+    get:
+      tags:
+        - roles
+      summary: Get a list of all the roles
+      description: |
+        ##### Permissions
+
+        `manage_system` permission is required.
+
+        __Minimum server version__: 5.33
+      responses:
+        "200":
+          description: Role retrieval successful
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Role"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+      x-code-samples:
+        - lang: Go
+          source: |
+            import "github.com/mattermost/mattermost-server/v5/model"
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            roles, resp := Client.GetAllRoles()
```

We can add a new line to separate the two doc.
```suggestion
            roles, resp := Client.GetAllRoles()

```